flexmeasures/data/models/generic_assets.py

```diff
@@ -498,6 +498,89 @@ def chart(
 
         return chart_specs
 
+    def multiple_chart(
```

Comment:
I don't like the name here (clashes a bit with "chart_for_multiple_sensors").

How about "charts()"?

Reply:
done.